Monkeypatch console.* calls on CI if we are asked to (#11897)
microsoft/vscode-python: news/3 Code Health/11896.md

> Monkeypatch `console.*` calls on CI if we are asked to.

Reviewer: ```suggestion
Monkeypatch `console.*` calls to the logger only in CI.
```

Author: I was wondering, should we be doing these only on CI or also on when running tests locally?

Reviewer: Unfortunately, in our code "CI" historically means "when running tests".  In this case It should actually mean "CI".  As far as I understand it, in CI (AzDO, etc.) the console isn't logged in a useful way.  When running tests locally we can see the console output.  If we really want to force logging console to a file when running tests locally then we only need to set the one environment variable.
